api/common/interfaces.go

```diff
@@ -668,6 +628,27 @@ func getSubscription(s Serverer, params map[string]interface{}) map[string]inter
 	})
 }
 
+func getRegistrant(s Serverer, params map[string]interface{}) map[string]interface{} {
+	if len(params) < 1 {
+		return respPacking(INVALID_PARAMS, "length of params is less than 1")
+	}
+
+	name, ok := params["name"].(string)
+	if !ok {
+		return respPacking(INVALID_PARAMS, "name should be a string")
+	}
+
+	registrant, expiresAt, err := chain.DefaultLedger.Store.GetRegistrant(name)
+	if err != nil {
+		return respPacking(INTERNAL_ERROR, err.Error())
+	}
+	reg := hex.EncodeToString(registrant)
+	response := make(map[string]interface{})
+	response["registrant"] = reg
+	response["expiresAt"] = expiresAt
```

Comment:
Suggestion:
```
response := map[string]interface{}{
        "registrant": reg,
        "expiresAt": expiresAt,
    }
```
Integrated allocation and assignment is friendly with compiler optimization.